### ISSUE TITLE
HFSS assign_impedance_to_sheet unjust False return

### ISSUE THREAD

**Issue by LarsWillocx** [ansys/pyaedt#5399]
### Before submitting the issue

- [x] I have searched among the existing issues
- [x] I am using a Python virtual environment

### Description of the bug

assign_impedance_to_sheet does not function in the eigenmode solution type for HFSS. No error message appears, only a False return. The exact same line of code does work for the standard solution type.

![Image](https://github.com/user-attachments/assets/f9bf99b4-9eee-4a23-8304-80029e59ca99)

### Steps To Reproduce

Create a design of the eigenmode solution type. Use assign_impedance_to_sheet.

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.10

### Installed packages

Package                      Version
---------------------------- ---------------
absl-py                      2.0.0
aiofiles                     22.1.0
aiosqlite                    0.19.0
annotated-types              0.7.0
ansys-pythonnet              3.1.0rc3
anyio                        3.6.2
argon2-cffi                  21.3.0
argon2-cffi-bindings         21.2.0
arrow                        1.2.3
asttokens                    2.2.1
astunparse                   1.6.3
async-lru                    2.0.4
attrs                        23.1.0
Babel                        2.12.1
backcall                     0.2.0
beautifulsoup4               4.12.2
bleach                       6.0.0
cachetools                   5.3.1
celluloid                    0.2.0
certifi                      2022.12.7
cffi                         1.15.1
charset-normalizer           3.1.0
click                        8.1.7
clr-loader                   0.2.6
cmake                        3.27.7
colorama                     0.4.6
comm                         0.1.3
contourpy                    1.0.7
cuda-python                  12.2.0
cupy-cuda12x                 12.1.0
cycler                       0.11.0
Cython                       3.0.0
debugpy                      1.6.7
decorator                    5.1.1
defusedxml                   0.7.1
derivative                   0.6.0
executing                    1.2.0
fastjsonschema               2.16.3
fastrlock                    0.8.1
ffmpeg                       1.4
ffmpeg-python                0.2.0
filelock                     3.13.1
flatbuffers                  23.5.26
fonttools                    4.39.3
fpdf2                        2.8.1
fqdn                         1.5.1
fsspec                       2023.10.0
future                       0.18.3
gast                         0.5.4
google-auth                  2.23.3
google-auth-oauthlib         1.0.0
google-pasta                 0.2.0
greenlet                     3.0.3
grpcio                       1.59.0
h11                          0.14.0
h5py                         3.10.0
httpcore                     1.0.6
httpx                        0.27.2
idna                         3.4
imageio                      2.32.0
imageio-ffmpeg               0.4.9
imbalanced-learn             0.11.0
imblearn                     0.0
ipykernel                    6.22.0
ipympl                       0.9.3
ipython                      8.12.0
ipython-genutils             0.2.0
ipywidgets                   8.0.6
isoduration                  20.11.0
jax                          0.4.20
jaxlib                       0.4.20
jedi                         0.18.2
Jinja2                       3.1.2
joblib                       1.3.2
json5                        0.9.11
jsonpointer                  2.3
jsonschema                   4.23.0
jsonschema-specifications    2024.10.1
julia                        0.6.1
jupyter                      1.0.0
jupyter_client               8.2.0
jupyter-console              6.6.3
jupyter_core                 5.3.0
jupyter-events               0.6.3
jupyter-lsp                  2.2.0
jupyter_server               2.5.0
jupyter_server_fileid        0.9.0
jupyter_server_terminals     0.4.4
jupyter_server_ydoc          0.8.0
jupyter-ydoc                 0.2.4
jupyterlab                   4.2.5
jupyterlab-pygments          0.2.2
jupyterlab_server            2.27.3
jupyterlab-widgets           3.0.7
keras                        2.14.0
kiwisolver                   1.4.4
libclang                     16.0.6
lxml                         5.2.1
Markdown                     3.5
MarkupSafe                   2.1.2
matplotlib                   3.7.1
matplotlib-inline            0.1.6
mistune                      2.0.5
ml-dtypes                    0.2.0
mpmath                       1.3.0
nbclassic                    0.5.5
nbclient                     0.7.3
nbconvert                    7.3.1
nbformat                     5.8.0
nest-asyncio                 1.5.6
networkx                     3.2.1
notebook                     6.5.4
notebook_shim                0.2.2
numpy                        1.24.2
oauthlib                     3.2.2
obspy                        1.4.0
opt-einsum                   3.3.0
packaging                    23.1
pandas                       2.1.1
pandocfilters                1.5.0
parso                        0.8.3
pickleshare                  0.7.5
Pillow                       9.5.0
pip                          24.2
pirate-get                   0.4.2
platformdirs                 3.2.0
plotly                       5.15.0
plumbum                      1.9.0
prometheus-client            0.16.0
prompt-toolkit               3.0.38
protobuf                     4.24.4
psutil                       5.9.5
pure-eval                    0.2.2
pyaedt                       0.11.1
pyasn1                       0.5.0
pyasn1-modules               0.3.0
pycparser                    2.21
pydantic                     2.9.2
pydantic_core                2.23.4
pyedb                        0.30.0
Pygments                     2.15.1
pyparsing                    3.0.9
pyperclip                    1.8.2
pyrsistent                   0.19.3
pysindy                      1.7.5
pysr                         0.16.3
python-dateutil              2.8.2
python-json-logger           2.0.7
pytomlpp                     1.0.13
pytz                         2023.3.post1
pywin32                      306
pywinpty                     2.0.10
PyYAML                       6.0
pyzmq                        25.0.2
qtconsole                    5.4.2
QtPy                         2.3.1
readligo                     0.6.1
referencing                  0.35.1
requests                     2.32.3
requests-oauthlib            1.3.1
rfc3339-validator            0.1.4
rfc3986-validator            0.1.1
rpds-py                      0.20.0
rpyc                         6.0.1
rsa                          4.9
Rtree                        1.3.0
schemdraw                    0.16
scikit-learn                 1.3.1
scikit-rf                    1.3.0
scipy                        1.10.1
scs                          3.2.3
Send2Trash                   1.8.0
setuptools                   68.0.0
six                          1.16.0
sniffio                      1.3.0
soupsieve                    2.4.1
SQLAlchemy                   2.0.29
stack-data                   0.6.2
sympy                        1.12
tenacity                     8.2.2
tensorboard                  2.14.1
tensorboard-data-server      0.7.1
tensorflow                   2.14.0
tensorflow-estimator         2.14.0
tensorflow-intel             2.14.0
tensorflow-io-gcs-filesystem 0.31.0
termcolor                    2.3.0
terminado                    0.17.1
threadpoolctl                3.2.0
tinycss2                     1.2.1
toml                         0.10.2
torch                        2.1.0+cu121
torch-cluster                1.6.3+pt21cu121
torch_geometric              2.4.0
torch-scatter                2.1.2
torch-sparse                 0.6.18
torch-spline-conv            1.2.2+pt21cu121
torchaudio                   2.1.0+cu121
torchvision                  0.16.0+cu121
tornado                      6.3
tqdm                         4.66.1
traitlets                    5.9.0
typing_extensions            4.8.0
tzdata                       2023.3
uri-template                 1.2.0
urllib3                      1.26.15
veryprettytable              0.8.1
wcwidth                      0.2.6
webcolors                    24.8.0
webencodings                 0.5.1
websocket-client             1.5.1
Werkzeug                     3.0.0
wheel                        0.41.2
widgetsnbextension           4.0.7
wrapt                        1.14.1
y-py                         0.5.9
ypy-websocket                0.8.2

**Comment by Samuelopez-ansys** [comment]
Hi @LarsWillocx ,

That was fixed one week ago in #5374.

It is already available in PyAEDT 0.11.3
